docs/quickstart.md

```diff
@@ -17,18 +17,18 @@ go mod init colx && touch main.go
 
 ## Import Dependencies
 
-First of all, you need to use `go get` to fetch the dependencies:
+First of all, you need to use `go get` to fetch the dependencies(for v4.0.2, the git hash is `3a18f1e`):
```

Comment:
What about give the approach of fetching the git hash?

Reply:
Done.